src/components/ListEmpty/ListEmpty.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './ListEmpty.css';
+
+const ListEmpty = () => {
```

Comment:
si no tenemos nada entre el `return`  y el `{` lo podriamos borrar para que quede de la siguiente manera:

```
const ListEmpty = () => (
  <>
      <p>Your shopping list is currently empty</p>
      <button>
        <Link className="link-button" to="/add-items">
          Add Item
        </Link>
      </button>
    </>
)
```

Reply:
we modified our function button onClick at commit **1c3d1988633471c32a4a34ecc7a90ace0200f444**